examples/multi_nic_common/example.tfvars

```diff
@@ -176,8 +176,24 @@ vmseries_common = {
   min_cpu_platform    = "Intel Cascade Lake"
   service_account_key = "sa-vmseries-01"
   bootstrap_options = {
+    # TODO: Modify the values below as per deployment requirements
     type                = "dhcp-client"
     mgmt-interface-swap = "enable"
+
+    # Panorama based bootstrap.
```

Comment:
Not very important but can we add 2 `#` here for the comment line ? I'm thinking someone will uncomment the section and that will throw out some errors.